platform/migration/platform-migration/src/test/java/org/codice/ddf/configuration/migration/ImportMigrationManagerImplTest.java

```diff
@@ -291,25 +302,13 @@ public void testConstructorWhenZipFileNotFound() throws Exception {
   }
 
   @Test
```

Comment:
❗️ Might want to re-add this test case but since we no longer have the input stream, we still need to test the failure scenario since the code can still be invoked for other errors